src/components/web-monitoring-ui.jsx

```diff
@@ -166,6 +167,7 @@ export default class WebMonitoringUi extends React.Component {
                 pages={this.state[this.state.pageFilter]}
               />
             }/>
+            <Route path="/version/:versionId" render={(routeProps) => <VersionRedirect api={api} {...routeProps} />} />
```

Comment:
Somehow I totally missed this before. `VersionRedirect` should be getting the API object via its `context`, not an explicit prop. Same as all the other routes.

Since you don’t need to add anything to it, I think you can use `component` instead of `render`, too:

```jsx
<Route path="/version/:versionId" component={VersionRedirect} />
```